router/fastdp.go

```diff
@@ -1171,6 +1179,45 @@ func (fastdp *FastDatapath) takeDecoder(lock *fastDatapathLock) *EthernetDecoder
 	return dec
 }
 
+// A checkFlowLoop checks whether the flow is free from the following loops:
+// * in_port == out_port, where in_port is non-vxlan vport;
+// * a packet is sent back to a vxlan tunnel it has been received from.
+func (fastdp *FastDatapath) checkFlowLoop(flow *odp.FlowSpec) bool {
```

Comment:
I think we should rename this function:

* It's not obvious from the name under which conditions it returns true (or false)
* The flowspec on its own doesn't contain a loop

How about `flowReflectsPackets`?